conditional/blueprints/attendance.py

```diff
@@ -1,37 +1,35 @@
 from datetime import datetime
 
 import structlog
-
 from flask import Blueprint, jsonify, redirect, request
 
-from conditional.util.ldap import ldap_get_current_students
-from conditional.util.ldap import ldap_get_active_members
-from conditional.util.ldap import ldap_is_eval_director
-from conditional.util.ldap import ldap_is_eboard
-from conditional.util.ldap import ldap_get_member
-
-from conditional.models.models import CurrentCoops
+from conditional import db, start_of_year, auth
```

Comment:
Move to style fixes commit.

Also why is this being reordered? (Alphabetization?)

Reply:
Based on: https://www.python.org/dev/peps/pep-0008/#imports
The lines being deleted are unused imports